### ISSUE TITLE
Setting Shell seems to have no effect

### ISSUE THREAD

**Issue by PetrKryslUCSD** [JunoLab/atom-julia-client#527]
I tried to set Shell to "C:\\Program Files\\Git\\bin\\bash.exe". It doesn't seem to have any effect, as Julia apparently starts in cmd.exe.

**Comment by pfitzseb** [comment]
True, this setting only determines what is started with the `Julia Client: New Terminal` command.

**Comment by PetrKryslUCSD** [comment]
So how do I control where Julia runs? I would like it to run in the bash, since it interacts much nicer with the operating and file system. Thanks.

**Comment by pfitzseb** [comment]
You can't right now -- we start the Julia executable directly, and that uses cmd by default.

**Comment by PetrKryslUCSD** [comment]
Where does this happen in the code, if I may ask?

**Comment by pfitzseb** [comment]
[Here](https://github.com/JunoLab/atom-julia-client/blob/master/lib/connection/process/basic2.js), and, if you use the powershell wrapper, [here](https://github.com/JunoLab/atom-julia-client/blob/master/script/spawnInterruptible.ps1). 
It should be fairly trivial to change the powershell script so it start `bash -e julia` instead, I think.

**Comment by PetrKryslUCSD** [comment]
I think it would be a nice enhancement. Thanks for giving it a thought.

**Comment by hkotkanen** [comment]
Hi, I'll comment here instead of making a new issue (can do that as well if you'd think it would be better) since it seems to be related: I'm also having trouble starting Julia and I'm trying to understand where the error message "/bin/sh: 1: julia: not found" is coming from. I'm on Ubuntu 18.04 and using zsh, where I the julia binary to my PATH in .zshrc, so it's understandable that /bin/sh can't find it.
However my $SHELL is '/bin/zsh', which is also shown as the default value on the settings page.

Everything works fine if I provide the absolute path to the binary in the settings, but I can't figure out where the /bin/sh is coming from. From node-pty-prebuilt?

**Comment by pfitzseb** [comment]
For both of your issues it would probably be easiest to just start Atom in the proper shell (or create a script that does that for you). I toyed around with implementing this but imho the additional code complexity isn't worth it.

@PetrKryslUCSD  On Windows you can e.g. call
```
"C:\Program Files\Git\bin\bash.exe" -c 'atom'
```
and things should work fine:
```
shell> which julia
/d/Programme/Julia-1.0.0/bin/julia

shell> where julia
D:\Programme\Julia-1.0.0\bin\julia.exe
```

Same goes for @hkotkanen -- just start Atom via
```
zsh -c atom
```
and Julia should be on your path.

**Comment by PetrKryslUCSD** [comment]
That is an excellent idea! Thanks.
